### PR TITLE
refactor: --unsafely-ignore-certificate-errors

### PR DESCRIPTION
**Pull request by bartlomieju**

### CODE REVIEW

File: cli/flags.rs [hunk 1]
```diff
@@ -217,7 +217,7 @@ impl Flags {
       _ => {}
     }
 
-    match &self.unsafely_treat_insecure_origin_as_secure {
+    match &self.unsafely_ignore_certificate_errors {
       Some(ic_allowlist) if ic_allowlist.is_empty() => {
         args.push("--unsafely-treat-insecure-origin-as-secure".to_string());
```
Comment (lucacasonato) [inline cli/flags.rs:222]: Missed this.

File: cli/program_state.rs [hunk 2]
```diff
@@ -119,15 +119,15 @@ impl ProgramState {
     }
 
     if let Some(insecure_allowlist) =
-      flags.unsafely_treat_insecure_origin_as_secure.as_ref()
+      flags.unsafely_ignore_certificate_errors.as_ref()
     {
       let domains = if insecure_allowlist.is_empty() {
         "for all domains".to_string()
```
Comment (lucacasonato) [inline cli/program_state.rs:125]:
```suggestion
        "for all hostnames".to_string()
```

File: cli/tests/cafile_ts_fetch_unsafe_ssl.ts.out [hunk 3]
```diff
@@ -1,2 +1,2 @@
-DANGER: SSL ceritificate validation is disabled for all domains
+DANGER: TLS ceritificate validation is disabled for all domains
```
Comment (lucacasonato) [inline cli/tests/cafile_ts_fetch_unsafe_ssl.ts.out:1]:
```suggestion
DANGER: TLS ceritificate validation is disabled for all hostnames
```

**Review by lucacasonato — APPROVED** [review]

File: cli/flags.rs [hunk 4]
```diff
@@ -217,13 +217,13 @@ impl Flags {
       _ => {}
     }
 
-    match &self.unsafely_treat_insecure_origin_as_secure {
+    match &self.unsafely_ignore_certificate_errors {
       Some(ic_allowlist) if ic_allowlist.is_empty() => {
-        args.push("--unsafely-treat-insecure-origin-as-secure".to_string());
+        args.push("--unsafely-ignore_certificate_errors".to_string());
```
Comment (chibat) [inline cli/flags.rs:222]: `--unsafely-ignore-certificate-errors` ?

File: cli/flags.rs [hunk 5]
```diff
@@ -217,13 +217,13 @@ impl Flags {
...
       }
       Some(ic_allowlist) => {
         let s = format!(
-          "--unsafely-treat-insecure-origin-as-secure={}",
+          "--unsafely-ignore_certificate_errors={}",
```
Comment (chibat) [inline cli/flags.rs:226]: `--unsafely-ignore-certificate-errors` ?

File: cli/flags.rs (hunk 4 above)
Comment (bartlomieju) [reply cli/flags.rs:222]: Thanks, fixed in https://github.com/denoland/deno/pull/11628

**Pull request merged**